src/functionalTest/resources/scenarios/RIA-8246-legal-rep-manage-fee-additional-payment-requested-notification.json

```diff
@@ -1,6 +1,7 @@
 {
   "description": "RIA-8246 Legal rep notification for Manage Fee Update - Additional payment requested",
   "launchDarklyKey": "dlrm-refund-feature-flag:true",
+  "launchDarklyKey": "dlrm-telephony-feature-flag:true",
```

Comment:
Can you have a look at CcdScenarioRunnerTest? 
It does not take multiple launchDarklyKey values. I'm not sure if the first or last is checked. You can debug locally, to see which value. (Will be good to know)

We can remove the refund one in this scenario.

Reply:
Removed refund flag. It takes the latest flag in CCDScenario